python/test/backend/test_device_backend.py

```diff
@@ -43,39 +41,7 @@ def build_for_backend(name, src, srcdir):
         scheme = 'posix_prefix'
     py_include_dir = sysconfig.get_paths(scheme=scheme)["include"]
 
-    ret = subprocess.check_call([cc, src, f"-I{py_include_dir}", f"-I{srcdir}", "-shared", "-fPIC", "-o", so])
-    if ret == 0:
-        return so
-    # fallback on setuptools
```

Comment:
If `ret != 0`, `subprocess` module will throw an exception and it turns out that this code branch is unreachable.